utils/testrunner/Matchers.js

```diff
@@ -26,99 +30,202 @@ module.exports = class Matchers {
     this._matchers[name] = matcher;
   }
 
-  expect(value) {
-    return new Expect(value, this._matchers);
+  expect(received) {
+    return new Expect(received, this._matchers);
   }
 };
 
+class MatchError extends Error {
+  constructor(message, formatter = null) {
+    super(message);
+    this.name = this.constructor.name;
+    this.formatter = formatter;
+    this.location = getLocation(__filename);
+    Error.captureStackTrace(this, this.constructor);
```

Comment:
What does this line do? Shouldn't calling super already capture a stack trace?

Reply:
That's to filter out the custom error constructor from the stack trace. That's a canonical way to create a custom error in node, e.g. see [here](https://dev.to/lvidakovic/custom-error-types-in-node-js-491a)

Reply:
It's not in firefox, which hurts the portability of the test runner, but I can probably polyfill it.